static/app/actionCreators/indicator.tsx

```diff
@@ -52,11 +64,6 @@ export function addMessage(
   ) {
     Sentry.captureException(new Error('Attempt to XHR response to Indicators'));
   }
-  if (type === 'undo' && typeof options.undo !== 'function') {
```

Comment:
I would lean towards preserving this check, and visibility over the bug if it were to come back.